apps/web/src/components/file-browser/file-browser-data.tsx

```
@@ -1,93 +1,366 @@
+								<tr
+									key={file.id}
+									className="hover:bg-accent/10 cursor-pointer border-t transition-colors"
+									onClick={() => handleRowClick(file)}
+									onKeyDown={e => {
+										if (e.key === "Enter" || e.key === " ") {
+											e.preventDefault();
+											handleRowClick(file);
+										}
+									}}
+									tabIndex={0}
+									role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)